src/EditorFeatures/CSharpTest/CodeActions/AddAwait/AddAwaitTests.cs

```diff
@@ -259,19 +260,75 @@ async Task<int> GetNumberAsync()
         }
 
         [Fact]
-        public async Task MissingOnSemiColon()
+        [WorkItem(35525, "https://github.com/dotnet/roslyn/issues/35525")]
+        public async Task OnSemiColon()
         {
-            await TestMissingInRegularAndScriptAsync(@"
+            await TestInRegularAndScriptAsync(@"
```

Comment:
Are these IDE changes intended in this PR? Seems unrelated

Reply:
It was based on a different PR, rebased to master for clarity.

Reply:
Didn't rebased correctly. WIP